tableauserverclient/server/request_options.py

```diff
@@ -25,6 +28,35 @@ class Direction:
         Desc = 'desc'
         Asc = 'asc'
 
+    class Builder:
+        def __init__(self):
+            self.__pagenumber = 1
```

Comment:
The double underscore was needed to disambiguate with the call to _pagenumber that I was specifically handling. If I don't do this, it actually fails